Update documentation on installing data apps (#950)
snowplow/documentation: docs/data-apps/index.md

## Accessing the Applications

> You can find the Data Applications in the `Applications` tab in the left sidebar of your Snowplow Console. If you have purchased a package with data applications available you can [install the application](#installing-applications) or you can click on the tile and request access. A Snowplow Customer Success Manager will then get in contact with you to discuss.

Reviewer: ```suggestion
You can find the Data Applications in the `Applications` tab in the left sidebar of your Snowplow Console. An application can be in one of three states:
* _By request_ — If you have not yet purchased a package that includes the application, you can click on “Learn more” and register your interest. A Snowplow Customer Success Manager will then get in contact with you to discuss getting access to the application.
* _Available_ — The application is ready to be set up. See the [installation instructions](#installing-applications) below.
* _Live_ — The application is ready to use.
```

Once the application is installed, clicking on the tile will launch the application in a separate browser tab. By default, anyone in your Console organization will be able to access data applications.

If you wish to invite others to use data applications but not have access to the rest of Console, you can [create a new user](/docs/using-the-snowplow-console/managing-users/adding-removing/index.md) and add a user with the `Data applications user` role. That user will then only see the Data Applications tab within Console. This permissions can be managed in the [usual way](/docs/using-the-snowplow-console/managing-users/managing-permissions/index.md).

## Installing applications

Once you have purchased an application you are able to install it. Click the tile to begin installing the application and follow the steps. 

> <img src={require("./images/install-app-tile.png").default} alt="Image of application tile showing start install process" />

Reviewer: ```suggestion
![Image of application tile showing start install process](images/install-app-tile.png)
```

Depending on the requirements of the application you may need to enable data models and enrichments in order to use it. These requirements are highlighted within the documentation pages for each data application and will also be shown to you during the install process. 

> <img src={require("./images/sample-app-install.png").default} alt="Image of application install process" />

Reviewer: Same as above

If you wish to invite others to use data applications but not have access to the rest of Console, you can [create a new user](/docs/using-the-snowplow-console/managing-users/adding-removing/index.md) and add a user with the `Data applications user` role. That user will then only see the Data Applications tab within Console. This permissions can be managed in the [usual way](/docs/using-the-snowplow-console/managing-users/managing-permissions/index.md).

## Installing applications

Once you have purchased an application you are able to install it. Click the tile to begin installing the application and follow the steps. 

![Image of application tile showing start install process](images/install-app-tile.png)

During installation you will need to [provide a connection](#application-connections) to the warehouse you would like the application to use. It will also highlight any pipeline and data model dependencies that are required.

![Image of application install process](images/sample-app-install.png)

> ## Dependencies on data models

Reviewer: ```suggestion
### Dependencies on data models
```

> Generally applications will depend on data models. If there are dependencies then the installation flow will highlight which models are required and what models you currently have running through console. It will also highlight any properties that need to be enabled on these data models. If you are running these models yourself outside of console then you will need to manually check the requirements for each application. These requirements are highlighted within the documentation pages for each application.

Reviewer: ```suggestion
Generally, applications will depend on data models. If there are dependencies, then the installation flow will highlight which models are required and what models you currently have [running via BDP](/docs/modeling-your-data/running-data-models-via-snowplow-bdp/standard-models/index.md). It will also highlight any properties that you need to enable or configure for these data models.

:::note
If you are running the necessary data models yourself outside of BDP, then you will need to manually check that your setup satisfies the requirements for each application. These requirements are listed within the documentation pages for each application.
:::
```

## Application connections

> To install a data application you will need to create a connection. This will be the location where your data model output or atomic table resides. Connections can be created during the application install process and involves securely sending credentials for the data app to use to fetch data.

Reviewer: ```suggestion
To install a data application, you will need to create a connection to where your data model output or atomic table resides. You can do this as part of the installation process. Creating a connection involves securely setting up credentials for the data app to use to fetch the data.

:::tip
You might have already set up a warehouse connection for loading and/or modeling the data. Data applications, however, require a different connection, as you will often want them to run under a different user/role and with different permissions.
:::
```

![Image of application install process](images/add-connection.png)

> Once you have selected a destination, provided the credentials and run any sql scripts we will then test the connection and once tested successfully will be available for the application to use.

Reviewer: ```suggestion
Once you have selected a destination, provided the credentials and run the suggested SQL script, BDP will test the connection. Upon a successful test, the application will be available to use.
```

* _By request_ — If you have not yet purchased a package that includes the application, you can click on “Learn more” and register your interest. A Snowplow Customer Success Manager will then get in contact with you to discuss getting access to the application.
* _Available_ — The application is ready to be set up. See the [installation instructions](#installing-applications) below.
* _Live_ — The application is ready to use.

Once the application is installed, clicking on the tile will launch the application in a separate browser tab. By default, anyone in your Console organization will be able to access data applications.

If you wish to invite others to use data applications but not have access to the rest of Console, you can [create a new user](/docs/using-the-snowplow-console/managing-users/adding-removing/index.md) and add a user with the `Data applications user` role. That user will then only see the Data Applications tab within Console. This permissions can be managed in the [usual way](/docs/using-the-snowplow-console/managing-users/managing-permissions/index.md).

## Installing applications

> Once you have purchased an application you are able to install it. Click the tile to begin installing the application and follow the steps. 

Reviewer: ```suggestion
Once you have purchased an application, you are able to install it. Click the tile to begin installing the application and follow the steps. 
```

![Image of application tile showing start install process](images/install-app-tile.png)

> During installation you will need to [provide a connection](#application-connections) to the warehouse you would like the application to use. It will also highlight any pipeline and data model dependencies that are required.

Reviewer: ```suggestion
During installation, you will need to [provide a connection](#application-connections) to the warehouse you would like the application to use. It will also highlight any pipeline and data model dependencies that are required.
```
